test/common/benchmark.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable required-modules */
+
+'use strict';
+
+const assert = require('assert');
+const fork = require('child_process').fork;
+const path = require('path');
+
+const runjs = path.join(__dirname, '..', '..', 'benchmark', 'run.js');
+
+function runBenchmark(name, args, env) {
+  const argv = [];
+
+  for (let i = 0; i < args.length; i++) {
+    argv.push('--set');
+    argv.push(args[i]);
```

Comment:
Nit: a for-of loop could be used here, and the argv.push lines can be coalesced.